packages/deno/src/generators/utils/testing/mock-app-generator.ts

```diff
@@ -0,0 +1,42 @@
+import { addProjectConfiguration, joinPathFragments, Tree } from '@nrwl/devkit';
+import type { DenoAppNormalizedSchema } from '../../application/schema';
+
+export function mockDenoApp(tree: Tree, opts: DenoAppNormalizedSchema) {
```

Comment:
Nitpick: Nothing is being mocked here, I'd change this to `createTestDenoApp` or `createDenoAppForTesting` or something similar.